saturday-afternoon/components/flights/Airlines.js

```diff
@@ -0,0 +1,55 @@
+// @flow
+import React, { Fragment } from "react";
+import { Col, Row, Avatar } from "antd";
+import { createFragmentContainer, graphql } from "react-relay";
+import idx from "idx";
+
+import type { Airlines as AirlinesType } from "./__generated__/Airlines.graphql";
+
+type Props = {
+  data: AirlinesType
+};
+
+const Airlines = ({ data: { legs } }: Props) => (
+  <Fragment>
+    {legs &&
```

Comment:
Honestly I'd rather use `const legs = data.legs || [];` then doing the `{data: {legs}}` and now the `legs && legs.map`

Reply:
sure